app/components/Account/AccountVoting.jsx

```diff
@@ -549,13 +552,73 @@ class AccountVoting extends React.Component {
         let workerArray = this._getWorkerArray();
 
         let voteThreshold = 0;
+        const hideProposals = filteredWorker => {
+            const isActive = worker => {
+                return (
```

Comment:
An account is considerd active when its within the workers list of line 615

Reply:
fixed